ui/src/app/components/help-sidebar.spec.tsx

```diff
@@ -360,66 +429,59 @@ describe('HelpSidebar', () => {
       accessLevel: WorkspaceAccessLevel.WRITER,
     });
 
-    const wrapper = await component();
+    component();
     expect(
-      wrapper.find({ 'data-test-id': 'help-sidebar-icon-apps' }).length
-    ).toBe(1);
+      await screen.findByTestId('help-sidebar-icon-apps')
+    ).toBeInTheDocument();
   });
 
   it('should display dynamic runtime status icon', async () => {
     setRuntimeStatus(RuntimeStatus.RUNNING);
-    const wrapper = await component();
-    await waitForFakeTimersAndUpdate(wrapper);
-
-    expect(runtimeStatusIcon(wrapper).prop('style').color).toEqual(
-      colors.asyncOperationStatus.running
-    );
+    component();
 
+    expect(
+      await findRuntimeStatusIcon(RuntimeStatus.RUNNING)
+    ).toBeInTheDocument();
     act(() => setRuntimeStatus(RuntimeStatus.DELETING));
-    await waitForFakeTimersAndUpdate(wrapper);
-
-    expect(runtimeStatusIcon(wrapper).prop('style').color).toEqual(
-      colors.asyncOperationStatus.stopping
-    );
+    expect(
+      await findRuntimeStatusIcon(RuntimeStatus.DELETING)
+    ).toBeInTheDocument();
 
     act(() => clearRuntime());
-    await waitForFakeTimersAndUpdate(wrapper);
-    runtimeStatusIcon(wrapper, /* exists */ false);
+
+    await waitForRuntimeStatusIconAbsence();
 
     act(() => setRuntimeStatus(RuntimeStatus.CREATING));
-    await waitForFakeTimersAndUpdate(wrapper);
-    expect(runtimeStatusIcon(wrapper).prop('style').color).toEqual(
-      colors.asyncOperationStatus.starting
-    );
+    expect(
+      await findRuntimeStatusIcon(RuntimeStatus.CREATING)
+    ).toBeInTheDocument();
   });
 
   it('should display "starting" UX during compound runtime op with no runtime', async () => {
     setRuntimeStatus(RuntimeStatus.DELETING);
     registerCompoundRuntimeOperation(workspaceDataStub.namespace, {
       aborter: new AbortController(),
     });
-    const wrapper = await component();
-    await waitForFakeTimersAndUpdate(wrapper);
+    component();
 
-    expect(runtimeStatusIcon(wrapper).prop('style').color).toEqual(
-      colors.asyncOperationStatus.stopping
-    );
+    expect(
+      await findRuntimeStatusIcon(RuntimeStatus.STOPPING)
```

Comment:
why STOPPING and not DELETING?

Reply:
I believe that it should be DELETING. I updated it. Thanks for pointing it out.